src/mono/wasm/debugger/BrowserDebugProxy/DebugStore.cs

```diff
@@ -1009,6 +1009,10 @@ public object ToScriptSource(int executionContextId, object executionContextAuxD
                 executionContextId,
                 executionContextAuxData,
                 //hash:  should be the v8 hash algo, managed implementation is pending
+                isLiveEdit = false,
```

Comment:
The changes here look unrelated?

Reply:
Yes, I checked and without sending this info the exceptions work as desired as well. Removing.